[Jetson] *** No rule to make target 'arch/aarch64/Makefile'.  Stop.

Hello!
I am trying to install the [ASUS AX1800 WiFi adapter](https://www.asus.com/networking-iot-servers/adapters/all-series/usb-ax56/) in an Nvidia Jetson Xavier NX developer card. It has the following specs:
Architecture: AARCH64
Kernel: 5.10.104-tegra. 
Operative system: Ubuntu 20.04 (focal)

I am encountering the following error when executing the `make` command:

```
ubuntu@ubuntu:~/drivers/rtl8852au$ make
#rm -f .symvers.8852au
make ARCH=aarch64 CROSS_COMPILE= -C /lib/modules/5.10.104-tegra/build M=/home/leolabs/drivers/rtl8852au  modules
make[1]: Entering directory '/usr/src/linux-headers-5.10.104-tegra-ubuntu20.04_aarch64/kernel-5.10'
Makefile:712: arch/aarch64/Makefile: No such file or directory
make[1]: *** No rule to make target 'arch/aarch64/Makefile'.  Stop.
make[1]: Leaving directory '/usr/src/linux-headers-5.10.104-tegra-ubuntu20.04_aarch64/kernel-5.10'
make: *** [Makefile:638: modules] Error 2
```

It might be related to [this issue](https://github.com/lwfinger/rtl8852au/issues/50).

Any advice is highly appreciated. 
Thank you in advance.

I am also get this Error
```
radxa@rock-5a:~/Downloads/rtl8852au$ make
make ARCH=aarch64 CROSS_COMPILE= -C /lib/modules/5.10.110-6-rockchip/build M=/home/radxa/Downloads/rtl8852au  modules
make[1]: Entering directory '/usr/src/linux-headers-5.10.110-6-rockchip'
Makefile:708: arch/aarch64/Makefile: No such file or directory
make[1]: *** No rule to make target 'arch/aarch64/Makefile'.  Stop.
make[1]: Leaving directory '/usr/src/linux-headers-5.10.110-6-rockchip'
make: *** [Makefile:636: modules] Error 2
```

```
radxa@rock-5a:~/Downloads/rtl8852au$ uname -m
aarch64
```

```
radxa@rock-5a:~/Downloads/rtl8852au$ uname -m | sed -e "s/i.86/i386/; s/ppc.*/powerpc/; s/armv.l/arm/; s/aarch64/arm64/;"
arm64
```

--- a/Makefile
+++ b/Makefile
@@ -629,6 +629,9 @@ else
 
 export CONFIG_RTL8852AU = m
 
+SUBARCH := $(shell uname -m | sed -e "s/i.86/i386/; s/ppc.*/powerpc/; s/armv.l/arm/; s/aarch64/arm64/;")
+ARCH ?= $(SUBARCH)
+
 all: modules
 
 modules:

--- a/os_dep/osdep_service_linux.c
+++ b/os_dep/osdep_service_linux.c
@@ -389,6 +389,8 @@ static int openFile(struct file **fpp, const char *path, int flag, int mode)
 {
 	struct file *fp;
 
+	MODULE_IMPORT_NS(VFS_internal_I_am_really_a_filesystem_and_am_NOT_a_driver);
+
 	fp = filp_open(path, flag, mode);
 	if (IS_ERR(fp)) {
 		*fpp = NULL;
@@ -502,6 +504,8 @@ static int isFileReadable(const char *path, u32 *sz)
 	mm_segment_t oldfs;
 	#endif
 	char buf;
+
+	MODULE_IMPORT_NS(VFS_internal_I_am_really_a_filesystem_and_am_NOT_a_driver);
 
 	fp = filp_open(path, O_RDONLY, 0);
 	if (IS_ERR(fp))

